templates/cli/lib/commands/push.js.twig

```diff
@@ -255,6 +256,19 @@ const pushResources = async ({ all, yes } = {}) => {
     }
 };
 
+const pushProject = async () => {
+    try {
+        await projectsUpdate({
+            projectId: localConfig.getProject().projectId,
+            name: localConfig.getProject().projectName,
+            parseOutput: false
```

Comment:
This should also push the project settings right?

Reply:
Yes. But pushing is done manually per service and auth

Reply:
The changes them self for pushing are in the other PR